### PR TITLE
Update recipes for new comestible tortilla

### PR DESCRIPTION
**Pull request by Cyrano7**
Update recipes for new comestible tortilla were it makes sense.

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -694,6 +694,7 @@
       [ "flour", 1 ],
       [ "cornmeal", 1 ],
       [ "flatbread", 1 ],
+	  [ "tortilla_corn", 1 ],
```
Comment (Darkyhard) [inline data/json/recipes/recipe_food.json]: Tabs here.

**Comment by Cyrano7** [comment]
@Darkyhard its soooo annoying missing just one tab. Github desktop doesn't recognize the difference between a tab and spaces so it won't let you update. So I have to go in the online editor to edit a super slow 11k line long file.

**Comment by Darkyhard** [comment]
You can use Notepad++ and change its' settings so that it will automatically replace all tabs by spaces.

**Comment by Cyrano7** [comment]
Ya I just did that now. Finally had enough haha.

**Pull request merged**